main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -344,6 +344,23 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
       }
     }
 
+    object Chars {
+      def Special: Rule1[String] = rule {
+        "\\\\" ~ push("\\") |
+          "\\'" ~ push("'") |
+          "\\\"" ~ push("\"") |
+          "\\n" ~ push("\n") |
+          "\\r" ~ push("\r") |
+          "\\t" ~ push("\t")
```

Comment:
Are we missing some escapes?:

https://stackoverflow.com/questions/1367322/what-are-all-the-escape-characters

Or perhaps its a deliberate choice only to have escapes for common/reasonable characters? What even is a backspace?

Reply:
Yeah it looks like there are a few archaic teletype things. I would guess that a `\b` or a `\f` is more likely a typo than an intentional escape sequence. They can still be accessed through `\uXXXX` so I think overall the best option is to include only the common ones.

Reply:
Agreed!